<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Idle</class>
 <widget class="QWidget" name="Idle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Idle Mode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Idle {
  background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #1a1a2e,
    stop:0.3 #16213e,
    stop:0.6 #0f3460,
    stop:1 #1a1a2e);
}

/* Step Card Normal Style */
.stepCard {
  font-family: 'Roboto Condensed', 'Arial', sans-serif;
  color: white;
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 rgba(11, 194, 0, 220),
    stop:0.5 rgba(11, 194, 0, 200),
    stop:1 rgba(8, 150, 0, 220));
  border: 3px solid rgba(255, 255, 255, 120);
  border-radius: 20px;
  padding: 25px;
  min-height: 220px;
}

/* Step Card Highlighted Style */
.stepCardHighlighted {
  font-family: 'Roboto Condensed', 'Arial', sans-serif;
  color: white;
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 rgba(11, 194, 0, 255),
    stop:0.5 rgba(15, 220, 0, 255),
    stop:1 rgba(11, 194, 0, 255));
  border: 5px solid rgba(255, 255, 255, 250);
  border-radius: 25px;
  padding: 30px;
  min-height: 240px;
}

/* Arrow Style */
.arrowLabel {
  font-size: 96px;
  color: #0bc200;
  background-color: transparent;
  font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="outerLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QLabel" name="titleLabel">
     <property name="minimumSize">
      <size>
       <width>603</width>
       <height>123</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>603</width>
       <height>123</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#titleLabel {
    color: #FFF;
    text-align: center;
    font-family: &quot;Roboto Condensed&quot;;
    font-style: italic;
    font-weight: 700;
    font-size: 55px;
    background-color: #0BC200;
    border-radius: 9px;
    border-bottom: 2px solid #020202;
}</string>
     </property>
     <property name="text">
      <string>How To Use</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="stepsLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
